lib/simplifyExpression/breakUpNumeratorSearch/index.js

```diff
@@ -27,8 +28,33 @@ function breakUpNumerator(node) {
   // At this point, we know that node is a fraction and its numerator is a sum
   // of terms that can't be collected or combined, so we should break it up.
   const fractionList = [];
-  const denominator = node.args[1];
-  numerator.args.forEach(arg => {
+  let denominator = node.args[1];
+
+  // Check if we can add a constant to make the fraction nicer
+  // fraction e.g. (2+x)/(5+x) -> (5+x)/(5+x) + 3/(5+x)
+  if (canFindDenominatorInNumerator(node)) {
+    let denominatorParenRemoved = false;
+    if (Node.Type.isParenthesis(denominator)) {
+      denominatorParenRemoved = true;
+      denominator = denominator.content;
+    }
+    const newNumerator = [];
+
+    // The constant value difference between the numerator and the denominator
+    const numeratorConstant = parseInt(numerator.args[1].value);
+    const denominatorConstant = parseInt(denominator.args[1].value);
+    const addedConstant = numeratorConstant - denominatorConstant;
+
+    newNumerator.push(Node.Creator.constant(addedConstant));
+    newNumerator.push(denominator);
+
+    numerator = newNumerator;
+
+    if (denominatorParenRemoved) {
+      denominator = Node.Creator.parenthesis(denominator);
+    }
+  }
```

Comment:
I think it'd be best if we can break this up into two steps. the step you added, and the break up fraction that comes after (and then maybe we can group in the cancelling that comes after). and then we'd have substeps for breaking up the fraction.

what do you think? (let me know if you want me to explain more what this means)